src/main/anovos/data_report/basic_report_generation.py

```diff
@@ -48,7 +48,7 @@ def anovos_basic_report(spark, idf, id_col='', label_col='', event_label='', out
     :param label_col: Label/Target column
     :param event_label: Value of (positive) event (i.e label 1)
     :param output_path: File Path for saving metrics and basic report
-    :param local_or_emr: "local" (default), "emr"
+    :param global_run_type: "local" (default), "emr"
                          "emr" if the files are read from or written in AWS s3
```

Comment:
@ranjanravish Can you please add a short documentation for the new "databricks" option? Thanks!

Reply:
Added short documentation for the new "databricks" option in docstring